server/methods/resetAvatar.js

```diff
@@ -26,6 +26,25 @@ Meteor.methods({
 			username: user.username,
 		});
 	},
+	resetAvatarAdmin(args) {
+		if (!args) {
+			throw new Meteor.Error('error-invalid-user', 'Invalid user', {
+				method: 'resetAvatarAdmin',
+			});
+		}
+
+		if (!settings.get('Accounts_AllowUserAvatarChange')) {
+			throw new Meteor.Error('error-not-allowed', 'Not allowed', {
+				method: 'resetAvatarAdmin',
+			});
+		}
+
+		FileUpload.getStore('Avatars').deleteByName(args.username);
```

Comment:
looks like there are no tests to unsure the called has the right permissions to allow resetting someone else avatar.

Reply:
let me update it.